glue/app/qt/application.py

```diff
@@ -362,6 +369,38 @@ def _setup_ui(self):
         self._log.resize(550, 550)
         self._log.hide()
 
+    def eventFilter(self, obj, event):
+        """
+        Mainly for handing Keyboard Shortcuts that are set
+        in keyboard_shortcuts.py
```

Comment:
I would reword this - maybe just say ``Mainly for handling Keyboard Shortcuts`` and leave the rest out, since user-defined shortcuts will get picked up too.